generic_abs_175.xml.fdm_material

```diff
@@ -11,6 +11,7 @@ Generic ABS 1.75mm profile. The data in this file may not be correct for your sp
         </name>
         <GUID>2780b345-577b-4a24-a2c5-12e6aad3e690</GUID>
         <version>11</version>
+        <bot_id>abs</bot_id>
```

Comment:
A material is not a bot_id. This is very confusing, I also do not know why you need it. The GUID uniquely identifies the material to do a look up in a database. 
Also when you would do this you also need to change the the fdmmaterial.xsd other wise digital factory throw an error.

Reply:
A bot_id _is_ not a material, but I do consider it to be a property of the material. And that is what metadata is; it contains useful properties regarding the material. 

> The GUID uniquely identifies the material to do a look up in a database.
According to this logic we can also remove for instance the color attribute from the metadata; as each guid uniquely identifies a color just lookup the color each time you try to retrieve it.